client/components/tinymce/index.jsx

```diff
@@ -63,7 +64,8 @@ import EditorHtmlToolbar from 'post-editor/editor-html-toolbar';
 	afterTheDeadlinePlugin,
 	wptextpatternPlugin,
 	toolbarPinPlugin,
-	embedReversalPlugin
+	embedReversalPlugin,
+	trackPaste
```

Comment:
let's also add in a trailing comma here. helps keep things consistent and diffs smaller/less confusing